client/src/components/Donation/donate-form.tsx

```diff
@@ -62,7 +62,7 @@ type PostCharge = (data: {
 type DonateFormProps = {
   postCharge: PostCharge;
   defaultTheme?: LocalStorageThemes;
-  email: string;
+  email?: string;
```

Comment:
This change points out something annoying:
- The client `Donation` type used by `/donate/...` requires `email`
- `/donate/charge-stripe-card` does not use `email`
- Our client is one accidental change away in the saga from causing the schema validation on the donate routes to fail; in fact, it might already, but I have not confirmed

Reply:
Okay, so it never was required. We just said it was.

I think it's better like this since it requires the component to handle this missing email (it already does).

We will need to be careful, but TS should help us a bit.